src/components/Card/Card.js

```diff
@@ -1,36 +1,101 @@
 import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
 import './Card.scss';
 
-export default class Card extends Component {
+class Card extends Component {
+  constructor() {
+    super();
+    this.state = {
+      optionBtn: false,
+      chooseSize: true,
+    };
+  }
+
+  btnHandler = () => {
```

Comment:
```suggestion
  openOptionBtn = () => {
```
이 이름이 더 직접적으로 함수의 역할을 설명할 수 있을 것 같습니다

Reply:
오오 감사합니다.

Reply:
오우 감사합니다!